odoo/tools/float_utils.py

```diff
@@ -90,6 +93,17 @@ def float_round(value, precision_digits=None, precision_rounding=None, rounding_
         normalized_value += sign*epsilon
         rounded_value = math.floor(abs(normalized_value)) * sign
 
+    # TIE-BREAKING: HALF-EVEN
+    # We want to apply HALF-EVEN tie-breaking rules, i.e. 0.5 rounds towards closest even number.
+    elif rounding_method == 'HALF-EVEN':
+        rounded_value = math.copysign(builtins.round(normalized_value), normalized_value)
+
+    # TIE-BREAKING: HALF-DOWN
+    # We want to apply HALF-DOWN tie-breaking rules, i.e. 0.5 rounds towards 0.
+    elif rounding_method == 'HALF-DOWN':
+        normalized_value -= math.copysign(epsilon, normalized_value)
+        rounded_value = round(normalized_value)
+
     # TIE-BREAKING: HALF-UP (for normal rounding)
     # We want to apply HALF-UP tie-breaking rules, i.e. 0.5 rounds away from 0.
     else:
```

Comment:
I'd personally want to do a `elif rounding_method == 'HALF-UP':` followed by a `else: raise ValueError`, but it should maybe not be done in stable?

Reply:
Ya, I agree ! Also would like that better.

Reply:
I can change it in the forward-port to master (but I do need this pr in 17.0).